src/Test/Runner/Log.elm

```diff
@@ -0,0 +1,76 @@
+module Test.Runner.Log exposing (run, runWithOptions)
+
+{-| # Log Runner
+
+Runs a test and outputs its results using `Debug.log`, then calls `Debug.crash`
+if there are any failures.
+
+This is not the prettiest runner, but it is simple and cross-platform. For
+example, you can use it as a crude Node runner like so:
+
+    $ elm-make LogRunnerExample.elm --output=elm.js
+    $ node elm.js
+
+This will log the test results to the console, then exit with exit code 0
+if the tests all passed, and 1 if any failed.
+
+@docs run, runWithOptions
+-}
+
+import Random
+import Test exposing (Test)
+import Test.Runner.String exposing (Summary)
+import String
+
+
+{-| Run the test using the default `Test.Runner.String` options.
+-}
+run : Test -> a -> a
```

Comment:
Ah, I think I get the intent then. Do I understand correctly that the proposed change is to let people call `Debug.log` themselves rather than encoding it into the API with a helper function?